### PR TITLE
Version Ticks for 7.2.0 RC Candidate

### PR DESCRIPTION
**Pull request by acpaquette**
Version ticks for releasing a new version of ISIS

## Description
Version ticks as defined in the ISIS semantic versioning for releasing a new version of ISIS (7.2.0)

## Related Issue
Q1 Support Sprint

## How Has This Been Validated?
Through Jenkins Pipelines

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: code.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "ISIS3",
     "organization": "U.S. Geological Survey",
     "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
-    "version": "7.0.0",
+    "version": "7.2.0",
```
Comment (Kelvinrr) [inline code.json:6]: Maybe I am missing something, but I am guessing the 7.1 tick was never ported over to dev?

**Review by Kelvinrr — COMMENTED** [review]
just one small question

File: code.json (hunk 1 above)
Comment (acpaquette) [reply code.json:6]: Nope, was done in the 7.1 tag

**Review by Kelvinrr — DISMISSED** [review]

**Review by Kelvinrr — DISMISSED** [review]

**Review by Kelvinrr — DISMISSED** [review]

**Review by Kelvinrr — APPROVED** [review]

**Comment by amystamile-usgs** [comment]
~~@acpaquette Do these changes also need to be pushed to dev? I am running into environment issues with GLIBC and not sure if these changes will fix it.~~

Nevermind, I see this was pushed to dev but only env changes to the meta.yaml.

**Pull request merged**